src/Parsers/tests/gtest_Parser.cpp

```diff
@@ -39,7 +39,9 @@ std::ostream & operator<<(std::ostream & ostr, const std::shared_ptr<IParser> pa
 
 std::ostream & operator<<(std::ostream & ostr, const ParserTestCase & test_case)
 {
-    return ostr << "ParserTestCase input: " << test_case.input_text;
+    std::string input_text{test_case.input_text};
```

Comment:
Maybe add a brief comment why newlines are removed? Just that future readers won't wonder.

Reply:
Done.